app/main.py

```diff
@@ -1,3 +1,80 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+
+        self.km = km
+
+    def __str__(self) -> str:
+
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (Distance, int)) -> Distance:
+
+        if isinstance(other, Distance):
+            other = other.km
+
+        new_km_amount = self.km + other
+
+        return Distance(new_km_amount)
+
+    def __iadd__(self, other: (Distance, int, float))\
+            -> (Distance, int, float):
```

Comment:
Invalid type hint for return value
```suggestion
    def __iadd__(self, other: (Distance, int, float)) -> Distance:
```